tianshou/env/venvs.py

```diff
@@ -383,11 +388,27 @@ class SubprocVectorEnv(BaseVectorEnv):
         Please refer to :class:`~tianshou.env.BaseVectorEnv` for other APIs' usage.
     """
 
-    def __init__(self, env_fns: Sequence[Callable[[], ENV_TYPE]], **kwargs: Any) -> None:
+    def __init__(
+        self,
+        env_fns: Sequence[Callable[[], ENV_TYPE]],
+        wait_num: int | None = None,
+        timeout: float | None = None,
+        share_memory: bool = False,
+        context: Literal["fork", "spawn"] | None = None,
+    ) -> None:
+        """:param share_memory: whether to share memory between the main process and the worker process.
+        :param context: the context to use for multiprocessing. It can be "fork" or "spawn".
```

Comment:
Mentioning the possible options is redundant given the type information. What we would need instead is guidance when to use which option.

Reply:
Tried to improve it